src/circuits/joinsplit.tcc

```diff
@@ -21,64 +20,64 @@ template<
 class joinsplit_gadget : libsnark::gadget<FieldT>
 {
 private:
-    // Multipacking gadgets for the inputs (nullifierS, commitmentS, val_pub_in,
-    // val_pub_out) (the root is a field element) `NumInputs + NumOutputs`
-    // because we pack the nullifiers (Inputs of JS = NumInputs), the
-    // commitments (Output of JS = NumOutputs) AND the v_pub_out taken out of
-    // the mix (+1) AND the public value v_pub_in that is put into the mix (+1)
-    // AND the signature hash h_sig (+1) AND the malleability tags h_iS (+
-    // NumInputs)
+    /*
+     * Multipacking gadgets for the inputs (nullifierS, commitmentS, val_pub_in,
+     *val_pub_out) (the root is a field element) `NumInputs + NumOutputs`
+     *because we pack the nullifiers (Inputs of JS = NumInputs), the commitments
+     *(Output of JS = NumOutputs) AND the v_pub_out taken out of the mix (+1)
+     *AND the public value v_pub_in that is put into the mix (+1) AND the
+     *signature hash h_sig (+1) AND the malleability tags h_iS (+ NumInputs)
+     **/
+    size_t bits = 2 * 64 + 3 * (1 + 2 * NumInputs + NumOutputs);
+    size_t log_r =
+        253; // log_2(21888242871839275222246405745257275088548364400416034343698204186575808495617);
```

Comment:
Comment above the line please to avoid such line break when running clang-format